src/connectdlg.cpp

```diff
@@ -42,31 +42,37 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
 
     // Add help text to controls -----------------------------------------------
     // directory
-    QString strDirectoryWT = "<b>" + tr ( "Directory" ) + ":</b> " +
-                             tr ( "Shows the servers listed by the selected directory. "
-                                  "You can add custom directories in Advanced Settings." );
-    QString strDirectoryAN = tr ( "Directory combo box" );
-
-    lblList->setWhatsThis ( strDirectoryWT );
-    lblList->setAccessibleName ( strDirectoryAN );
+    QString strDirectoryWT = "<b>" + tr ( "Directory" ) + ":</b> " + tr ( "The list of available directories." ) + "<br/>" +
+                             tr ( "Select a directory to list the servers registered with it." ) + "<br/>" +
+                             tr ( "You can add custom directories in Advanced Settings." );
```

Comment:
I don't see **Advanced Settings**. I see **Advanced Setup**, a tab in the **Settings** dialog.